### PR TITLE
fix(binaries): add binary build targets name column

### PR DESCRIPTION
**Pull request by steebchen**
[rendered diff](https://github.com/prisma/specs/pull/357/files?short_path=cbdeb0c#diff-cbdeb0c86b8f504fb01b42c8fd06bfe2)
[rendered view](https://github.com/prisma/specs/blob/fix/binaries/build-target-name-1/binaries/Readme.md#binary-build-targets)

### CODE REVIEW

**Comment by janpio** [comment]
What does "Name" mean exactly here? Filename?

**Comment by steebchen** [comment]
It's probably more like a "codename", and it also maps to the value you need to specify in your prisma.schema  if you need multiple binaries.

File: binaries/Readme.md [hunk 1]
```diff
@@ -177,14 +177,14 @@ Notes
...
-| `rhel`         | CentOS 6            | 1.0.x              | [prisma](https://prisma-builds.s3-eu-west-1.amazonaws.com/master/latest/rhel-openssl-1.0.x/prisma.gz)   [migration-engine](https://prisma-builds.s3-eu-west-1.amazonaws.com/master/latest/rhel-openssl-1.0.x/migration-engine.gz)   [introspection-engine](https://prisma-builds.s3-eu-west-1.amazonaws.com/master/latest/rhel-openssl-1.0.x/introspection-engine.gz)   [prisma-fmt](https://prisma-builds.s3-eu-west-1.amazonaws.com/master/latest/rhel-openssl-1.0.x/prisma-fmt.gz)   |
-|                |                     | 1.1.x              | [prisma](https://prisma-builds.s3-eu-west-1.amazonaws.com/master/latest/rhel-openssl-1.1.x/prisma.gz)   [migration-engine](https://prisma-builds.s3-eu-west-1.amazonaws.com/master/latest/rhel-openssl-1.1.x/migration-engine.gz)   [introspection-engine](https://prisma-builds.s3-eu-west-1.amazonaws.com/master/latest/rhel-openssl-1.1.x/introspection-engine.gz)   [prisma-fmt](https://prisma-builds.s3-eu-west-1.amazonaws.com/master/latest/rhel-openssl-1.1.x/prisma-fmt.gz)   |
+| Build                  | Build OS            | OpenSSL | Downloads |
```
Comment (janpio) [inline binaries/Readme.md]: Maybe:
```suggestion
| Build                  | Build OS            | OpenSSL | Downloads (`.gz`) |
```

**Comment by janpio** [comment]
Ok, then I like the current state with "Build" better. Could also be "Build name" if you want.

**Comment by steebchen** [comment]
Did you notice the new changes? It looks as follows at the momemnt, do you think that's good?

image

**Comment by janpio** [comment]
Yes I saw that. I suggested "Build name" because it incorporates you initial goal of adding a "name" ;)

**Comment by janpio** [comment]
HTML list would look like this btw:
image

**Comment by steebchen** [comment]
changed to "Build name". I'm ready to merge if you're fine with this, I don't think the html thing looks better (it will get huge when each build takes so much space)

**Pull request merged**